Mlem/Views/Shared/Posts/Post Sizes/Large Post.swift

```diff
@@ -33,6 +34,7 @@ struct LargePost: View {
         self.postView = postView
         self.isExpanded = isExpanded
         self.maxHeight = isExpanded ? .infinity : AppConstants.maxFeedPostHeight
+        self.titleColor = !isExpanded && postView.read ? Color.secondary : Color.primary
```

Comment:
nit, I think we should make this value a computed property

Reply:
Resolved in latest commit